arm/Microsoft.Network/virtualNetworks/readme.md

```diff
@@ -154,8 +154,8 @@ The network security group and route table resources must reside in the same res
 
 | Output Name | Type | Description |
 | :-- | :-- | :-- |
-| `subnetResourceIds` | array | The resource IDs of the deployed subnets |
-| `subnetNames` | array | The names of the deployed subnets |
+| `subnetNames` | array | The resource IDs of the deployed subnets |
+| `subnetResourceIds` | array | The names of the deployed subnets |
```

Comment:
Something is wrong here. subnetNames = resource IDs, and subnetResourceIDs = names? :D

Reply:
Fixed